backend/assignments/1/src/main/java/org/example/Transaction.java

```diff
@@ -0,0 +1,85 @@
+package org.example;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+
+enum TransactionType {
```

Comment:
good, now extract this enum into a different file

Reply:
Expected more modularity, better folder structures, try adding design patterns.
Which design pattern can easily to used in this question for a small subproblem we solved